lammps-tutorials.tex

```diff
@@ -4212,16 +4225,14 @@ \subsubsection{Simulating the reaction}
 \end{figure}
 
 \section*{Author Contributions}
-% S.G. to update
-S.G. conceived and wrote all the online tutorials and underlying Sphinx documentation
-for \href{https://lammpstutorials.github.io}{lammpstutorials.github.io}.
-
-\noindent A.K. wrote the LAMMPS GUI software and helped revise the
-tutorials for use with it.
 
-\noindent J.G. is the principal author of fix bond/react and type label
-support in LAMMPS.  He revised tutorials to use type labels.
-% and contributed the eighth tutorial using fix bond/react (?).
+S.G. conceived and wrote the original online tutorials and underlying Sphinx documentation
```

Comment:
@jrgissing @akohlmey I revised the author contribution part to add the tutorial 8 to J.G.'s contrib.  If you don't find the decription fair, please feel free to update.